das-cli/src/commands/query_agent/query_agent_cli.py

```
@@ -229,7 +229,7 @@ def run(self, port_range: str):
 class QueryAgentCli(CommandGroup):
     name = "query-agent"
 
-    aliases = ["qa", "query"]
+    aliases = ["qa", "query", "query-engine", "qe"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)